### ISSUE TITLE
Wrong colors in the Anilist. Light Latte, Red.

### ISSUE THREAD

**Issue by swooyuk** [catppuccin/userstyles#1457]
### Is there an existing issue outlining your problem?

- [X] I am using the latest version of the provided userstyle.
- [X] I have searched the existing issues and they do not solve my problem.

### What userstyles are you seeing the problem on?

lbl:anilist

### Describe your problem.

all the colors have an error in the name of the work

### Attach screenshots.

![Captura de tela 2024-11-15 142534](https://github.com/user-attachments/assets/f69dcf90-d0ff-4480-85ee-f29c63187fef)

### What browser(s) are you seeing the problem on?

Google Chrome 131.0.6778.69

### Any additional comments?

I'm using Catppuccin Latte with a Red accent color.

**Comment by AnubisNekhet** [comment]
can you provide a URL to the page with the styling error?

**Comment by AnubisNekhet** [comment]
Are you sure that's not just your profile color?

**Comment by swooyuk** [comment]
yes, I'm sure, I've tested it with all the colors and they look the same

**Comment by AnubisNekhet** [comment]
Can you provide a URL to the page with the issue?

**Comment by swooyuk** [comment]
https://anilist.co/user/floresier/animelist
https://anilist.co/user/floresier/mangalist

it's having problems only in the two lists and in all the users I enter on the site as well
